java/org.eclipse.set.toolboxmodel.releng.target/org.eclipse.set.toolboxmodel.releng.target.target

```diff
@@ -1,65 +1,80 @@
-			  </dependency>
-			  <dependency>
-				  <groupId>org.eclipse.set</groupId>
-				  <artifactId>org.eclipse.set.model.model11001.editor</artifactId>
-				  <version>2.3.0-SNAPSHOT</version>
-				  <type>jar</type>
-			  </dependency>
-			  <dependency>
-				  <groupId>org.eclipse.set</groupId>
-				  <artifactId>org.eclipse.set.model.model11001</artifactId>
-				  <version>2.3.0-SNAPSHOT</version>
-				  <type>jar</type>
-			  </dependency>
-			  <dependency>
-				  <groupId>org.eclipse.set</groupId>
-				  <artifactId>org.eclipse.set.model.modelservice</artifactId>
-				  <version>2.3.0-SNAPSHOT</version>
-				  <type>jar</type>
-			  </dependency>
-		  </dependencies>
-		  <repositories>
-			  <repository>
-				  <id>set-github</id>
-				  <url>https://maven.pkg.github.com/eclipse-set/model</url>
-			  </repository>
-		  </repositories>
-	  </location>
-  </locations>
-</target>
\ No newline at end of file
+	<locations>
+		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
+			<unit id="org.eclipse.sdk.feature.group" version="4.30.0.v20231201-0512"/>
+			<unit id="org.eclipse.jdt.feature.group" version="3.19.300.v20231201-0110"/>
+			<unit id="org.eclipse.emf.ecore.feature.group" version="2.36.0.v20231002-1156"/>
+			<unit id="org.eclipse.emf.sdk.feature.group" version="2.36.0.v20231107-0612"/>
+			<unit id="org.eclipse.e4.rcp.feature.group" version="4.30.0.v20231113-1355"/>
+			<unit id="org.eclipse.net4j.util.feature.group" version="4.22.0.v20231129-1234"/>
+			<unit id="org.eclipse.equinox.sdk.feature.group" version="3.23.1000.v20231114-0937"/>
+			<unit id="junit-jupiter-api" version="5.10.1"/>
+			<unit id="junit-jupiter-engine" version="5.10.1"/>
+			<unit id="org.apache.commons.commons-io" version="2.15.0"/>
+			<unit id="org.apache.commons.jxpath" version="1.3.0.v200911051830"/>
+			<unit id="com.ibm.icu" version="74.1.0"/>
+			<unit id="com.sun.jna" version="5.13.0.v20230812-1000"/>
+			<unit id="com.sun.jna.platform" version="5.13.0"/>
+			<unit id="bcpg" version="1.77.0"/>
+			<unit id="bcprov" version="1.77.0"/>
+			<unit id="jakarta.annotation-api" version="2.1.1"/>
+			<unit id="jakarta.inject.jakarta.inject-api" version="2.0.1"/>
+			<unit id="com.google.guava" version="32.1.3.jre"/>
+			<unit id="com.google.guava.failureaccess" version="1.0.2"/>
+			<unit id="org.apache.batik.css" version="1.17.0.v20231009-1000"/>
+			<unit id="org.apache.batik.constants" version="1.17.0.v20231009-1000"/>
+			<unit id="org.apache.batik.i18n" version="1.17.0.v20231009-1000"/>
+			<unit id="org.apache.batik.util" version="1.17.0.v20231009-1000"/>
+			<unit id="org.eclipse.orbit.xml-apis-ext" version="1.0.0.v20230923-0644"/>
+			<repository location="https://download.eclipse.org/releases/2023-12/"/>
+		</location>
+		
+		<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
```

Comment:
Xtend 2.34 is released and should be used instead

Reply:
ok